File: BMI.py
#Created by Subhadeep Mandal
weight = int(input("Enter weight in kg\n"))
height = float(input("Enter height  in metre\n"))

BMI=weight/(height*height)

if BMI < 18.5:
    print ("Underweight")
elif BMI >=18.5 and BMI<25:
    print ("Normal")
elif BMI >=25 and BMI<30:
    print ("Overweight")
elif BMI>30:
    print ("Obesity")

File: Java Fundamentals programs/BMI.java
//Java my A$$

import java.util.Scanner;

public class BMI {
    public static void main(String[] args){

        Scanner input = new Scanner(System.in);

        System.out.println("BMI VALUES");
        System.out.println("Underweight: less than 18.5");
        System.out.println("Normal: between 18.5 and 24.9");
        System.out.println("Overweight: between 25 and 29.9");
        System.out.println("Obese: above 30\n");

        double weight, height;
        double BMI;
        System.out.println("Enter Your Weight:");
        weight=input.nextFloat();
        System.out.println("Enter Your Height:");
        height=input.nextFloat();
        BMI=weight/(height*height);

        System.out.println("BMI is: " + BMI);
    }
}